TestApp/android/app/src/main/java/com/testapp/TestAppNativeModule.java

```diff
@@ -95,4 +98,16 @@ public void run() {
             }
         });
     }
+
+    @ReactMethod
+    public void saveSessionTrackerState(boolean state) {
+        mSharedPreferences.edit()
+                .putBoolean(SESSION_TRACKER_ENABLED_KEY, state)
+                .apply();
+    }
+
+    @ReactMethod
+    public void getSessionTrackerState(Promise promise) {
+        promise.resolve(mSharedPreferences.getBoolean(SESSION_TRACKER_ENABLED_KEY, false)? 1:0);
```

Comment:
formatting

Reply:
it should be
`promise.resolve(mSharedPreferences.getBoolean(SESSION_TRACKER_ENABLED_KEY, false) ? 1 : 0);`